packages/viewer/src/index.ts

```diff
@@ -2,9 +2,12 @@
  * @module Viewer
  */
 import * as ColorMaterial from './lib/scenes/colorMaterial';
+import * as TransformationDelta from './lib/transforms/transformation-delta';
```

Comment:
Could we consider adding an `index.ts` file to the `./lib/transforms` directory and exporting this there? I think we'll also want to export the `TransformController`, so it'd be nice to have an index file there